lib/browser/rpc.js

```diff
@@ -88,38 +88,37 @@ export function asyncOpenRealm(id, config, callback) {
     }));
 }
 
-export function _anonymousRPC(args) {
-    args = args.map((arg) => serialize(null, arg));
-    const result = sendRequest('_anonymous', { arguments: args });
+export function _anonymousRPC() {
+    const result = sendRequest('_anonymous', { arguments: undefined });
     return deserialize(undefined, result);
 }
 
-export function _appleRPC(args) {
-    args = args.map((arg) => serialize(null, arg));
+export function _appleRPC(token) {
+    const args = Array.prototype.map.call(arguments, (arg) => serialize(null, arg));
     const result = sendRequest('_apple', { arguments: args });
     return deserialize(undefined, result);
 }
 
-export function _emailPasswordRPC(args) {
-    args = args.map((arg) => serialize(null, arg));
+export function _emailPasswordRPC(email, password) {
```

Comment:
I suppose these positional arguments will become unused variables. Are there any reason for naming them explicitly instead of simply removing them?

Reply:
any method regardless if public or not should name the arguments.